addons/G36_mag_fix/script_component.hpp

```diff
@@ -0,0 +1,14 @@
+#define COMPONENT ugl_fix
```

Comment:
Hier passt was nicht, müsste das nicht addons/ugl_fix sein?

Reply:
habs mit in den RHS folder